src/software/ai/hl/stp/tactic/pass_defender/pass_defender_tactic.cpp

```diff
@@ -30,9 +33,22 @@ void PassDefenderTactic::updatePrimitive(const TacticUpdate &tactic_update,
 {
     if (reset_fsm)
     {
-        fsm_map[tactic_update.robot.id()] =
-            std::make_unique<FSM<PassDefenderFSM>>(PassDefenderFSM());
+        fsm_map[tactic_update.robot.id()] = std::make_unique<FSM<PassDefenderFSM>>(
+                PassDefenderFSM(),
+                DribbleFSM(ai_config.dribble_tactic_config()));
     }
+    Point ball_position       = tactic_update.world_ptr->ball().position();
+    Point enemy_goal_center   = tactic_update.world_ptr->field().enemyGoal().centre();
+    Vector ball_to_net_vector = Vector(enemy_goal_center.x() - ball_position.x(),
+                                       enemy_goal_center.y() - ball_position.y());
+    DribbleFSM::ControlParams dribble_control_params{
+            .dribble_destination       = ball_position,
+            .final_dribble_orientation = ball_to_net_vector.orientation(),
+            .allow_excessive_dribbling = false};
+
+
     fsm_map.at(tactic_update.robot.id())
         ->process_event(PassDefenderFSM::Update(control_params, tactic_update));
+    fsm_map.at(tactic_update.robot.id())
+            ->process_event(DribbleFSM::Update(dribble_control_params, tactic_update));
```

Comment:
I think with william's suggestions, we may not need this: https://github.com/UBC-Thunderbots/Software/pull/3230#discussion_r1667770230